docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,26 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+From Maven
+
+    <dependencies>
```

Comment:
It's best to call out the language explicitly here so that we get syntax highlighting:
```asciidoc
[source,xml]
----
<dependencies>
    <dependency>
      <groupId>org.ehcache</groupId>
      <artifactId>ehcache</artifactId>
      <version>3.10.0</version>
    </dependency>
</dependencies>
----
```

Reply:
Accommodated this review comment and push the changes